diff -up samtools-0.1.5c/misc/Makefile.compilefix samtools-0.1.5c/misc/Makefile
--- samtools-0.1.5c/misc/Makefile.compilefix	2009-07-11 15:33:04.000000000 +0200
+++ samtools-0.1.5c/misc/Makefile	2009-07-11 15:33:22.000000000 +0200
@@ -1,6 +1,6 @@
 CC=			gcc
 CXX=		g++
-CFLAGS=		-g -Wall -O2 -m64 #-arch ppc
+CFLAGS=		-g -Wall -O2 #-m64 #-arch ppc
 CXXFLAGS=	$(CFLAGS)
 DFLAGS=		-D_FILE_OFFSET_BITS=64
 OBJS=		
